Sources/XcbeautifyLib/CaptureGroups.swift

```diff
@@ -1866,3 +1866,24 @@ struct SwiftDriverJobDiscoveryCompilingCaptureGroup: CaptureGroup {
         self.project = project
     }
 }
+
+/// This output is printed when running
+/// `xcodebuild test -scheme xcbeautify-Package -destination 'platform=macOS,arch=arm64'`.
+struct NoteCaptureGroup: CaptureGroup {
+    static let outputType: OutputType = .task
+
+    /// Regular expression captured groups:
+    /// $1 = note
+    /// $2 = information
+    static let regex = Regex(pattern: "^(note:) (.*)")
```

Comment:
nit: We can change this to `"^note: (.*)$".

No need to capture `note`, since it's a static keyword.